src/components/Layouts/adminLayout.jsx

```
@@ -0,0 +1,179 @@
+import line from "../../assets/adminPage/Line 256.svg";
+import star from "../../assets/adminPage/star.png";
+import clients from "../../assets/adminPage/clients.svg";
+import settings from "../../assets/adminPage/settings.svg";
+import home from "../../assets/adminPage/home.svg";
+import hand from "../../assets/adminPage/hand-holding-heart.svg";
+import users from "../../assets/adminPage/users.svg";
+import burger from "../../assets/adminPage/burger.svg";
+
+const AdminLayout = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [asideIsOpen, setAsideIsOpen] = useState(false);
```

ESLint

[error] 'asideIsOpen' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)